test/integration/features/provision_earier_version.feature

```diff
@@ -0,0 +1,42 @@
+@minishift-only
```

Comment:
@praveenkumar There is a typo in filename. Deployment of an application is an open question as it adds time to execution of this feature, but I would suggest to go for it.

Rest looks fine to me, big +1 for adding this test as part of PR.